store/index.ts

```diff
@@ -17,53 +18,54 @@ function delay(ms: number) {
 export const state = (): RootState => ({
   loading: false,
   userAuthToken: undefined,
-    isLoggedIn: false
+  isLoggedIn: false
 })
 
 export const mutations: MutationTree<RootState> = {
   setLoading(state, b: boolean) {
     state.loading = b
   },
   setUserAuthToken(state, authToken: string) {
-      if (authToken == null) {
-          state.userAuthToken = undefined
-          state.isLoggedIn = false
-      } else {
-          state.isLoggedIn = true
-          state.userAuthToken = authToken
-      }
+    if(!authToken) {
+        state.isLoggedIn = false
+    } else if(authToken === state.userAuthToken) {
+        return
+    } else {
+        state.isLoggedIn = true
+        state.userAuthToken = authToken
+        axios.defaults.headers.common['authToken'] = authToken
+        Vue.$cookies.set('vmt-authToken', authToken)
+    }
   }
 }
 
 export const actions: ActionTree<RootState, RootState> = {
   async authenticate({ commit }, login: { email: string; password: string }) {
     commit('setLoading', true)
-    return axios
```

Comment:
I think we need to set isLoggedIn to be false here to fix the access to add-vehicles page bug